docs/development.md

```diff
@@ -27,7 +27,14 @@ Follow the [instal](install.md) steps to prepare the source code. Then follow th
 
 ### Visual Studio Code
 
-First, click on the Python version at the bottom left and enter the path where the above command was issued. This will point the Code to the Poetry virtual environment.
+First, click on the Python version at the bottom left of the editor's
+window and enter the path where the above command was issued. You
```

Comment:
Can it be put in troubleshooting? It's not met in every setup.

Reply:
I don't get it. If you do what I added, then VSCode seems to start guessing right every next time you open it, so it is a setup instruction. You move it after we merge, if it pleases you, OK?

Reply:
The poetry bases on current folder to provide different environments. My understanding is if we get current Python path right, the poetry can make its venv path right. So if provide poetry venv settings here, it may be confusing which Python underlying the poetry.

I'm using poetry with anaconda together. I just need to create an env in anaconda, and poetry will leverage that folder, and no other venv created. It doesn't need to put the venv path into settings.

So I hope this part can be moved to troubleshooting part, let users pick it up based on what problem they meet. If you don't have bandwidth on this part. Can you describe what problem you met, and put the content into an issue? I will update this part later.

Reply:
Well I'm not attached to this piece of text, so be it. But we'd better stop practising things and documenting others, like using conda + poetry, no? Are you doing this https://github.com/python-poetry/poetry/pull/1432 ? Try using it with poetry and vscode only and see if it can guess your venv everytime you switch the workplace :) The text is gone now.